triso.ipynb

```diff
@@ -304,12 +305,31 @@
     "\n",
     "settings = openmc.Settings()\n",
     "settings.run_mode = 'plot'\n",
+    "settings.source_mesh = None\n",
```

Comment:
Is this a secret feature I don't know about?! :exploding_head: In all seriousness, it looks like you have a bunch of local changes to this notebook that crept in (and need reverting)

Reply:
Haha my bad. Thanks for catching that! I've reverted this notebook and re-ran it just to make sure it all went smoothly.